core/src/main/java/org/jenkins/ui/icon/BuildStatusIcon.java

```diff
@@ -0,0 +1,27 @@
+package org.jenkins.ui.icon;
+
+public class BuildStatusIcon extends Icon {
```

Comment:
Shall we make it `@Restricted`?

Reply:
I think we should unless there is a reason not to

Reply:
I'll do it then!